azure_li_services/units/network.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2017 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of azure-li-services.
+#
+# azure-li-services is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# azure-li-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with azure-li-services.  If not, see <http://www.gnu.org/licenses/>
+#
+import ipaddress
+from textwrap import dedent
+
+# project
+from azure_li_services.runtime_config import RuntimeConfig
+from azure_li_services.defaults import Defaults
+from azure_li_services.exceptions import AzureLiNetworkConfigDataException
+
+
+def main():
+    """
+    Azure Li/Vli network setup
+
+    Creates network configuration files to successfully start the
+    network in the scope of an Azure Li/Vli instance
```

Comment:
Note that VLI is going to be different. It's using bridged ethernet ports. I was going to address that when we got there, but if you want it sooner, I can provide that.

I'd like to focus on just LI at this point, dealing with VLI after LI is working.